.gitignore

```diff
@@ -13,3 +13,5 @@ wandb
 *.egg*
 __pycache__
 build/
+
+data
```

Comment:
This needs to be removed. There's a `dance/data` module. Ignoring `data` will cause problems tracking changes in the `dance.data` module

Reply:
changed